### ISSUE TITLE
Anamnesis Silently Fails To Launch

### ISSUE THREAD

**Issue by SnowKatie** [imchillin/Anamnesis#1134]
**Describe the bug**
Upon launching the Anamnesis.exe nothing happens. Task manager shows to progresses that begin both of which immediately proceed to die.

**To Reproduce**
Steps to reproduce the behaviour:
1. Run the Anamnesis.exe

**Expected behaviour**
For something to boot, settings to open,

**Log file**
No log file is produced in AppData, no Anamnesis file exists.

**Current System Information**

- RUNTIME: Microsoft Windows Desktop Runtime - 6.0.4 (x64)
- AV: ESET NOD32 
- Windows 11, 64x bit OS.
- Anamnesis version 2022-05-31
- FF XIV running Directx 11

**What have I tried**

- Completely disable ESET NOD32
- Add exemptions for ESET NOD32 and keep it running
- Use Runtime versions 6.0.5, 6.0.6, 5.0.15 all with fresh reboots and installations.
- Run Anamnesis both in C:/ and the games native drive E:/ 
- Running the program a million times
- Program always runs as administrator
- FF XIV is always booted and ready on launch
- Installing Anamnesis only after ESET NOD32 is disabled to prevent corruption

**Comment by Ani-ki** [comment]
We may need to take a look at your Event Viewer to see if that can shed some light on it.

**Comment by SnowKatie** [comment]
Hello @Ani-ki,

After you notifying me to look through Event Viewer I did so and found the exemption causing the program to crash and fixed it myself. It was some sort of clash with another application.

The application causing Anamnesis to crash was: **Rivatuner Statistics Viewer (RTSS)**. A program used whenever you want to use MSI AFTERBURNER to see your FPS in games.

After I went to my tray and stopped RTSS from running, Anamensis opened without issue.



`Faulting application name: Anamnesis.exe, version: 1.0.1.0, time stamp: 0x62571213
Faulting module name: RTSSHooks64.dll, version: 0.0.0.0, time stamp: 0x5aaa3d86
Exception code: 0xc0000005
Fault offset: 0x0000000000014fb0
Faulting process ID: 0x418
Faulting application start time: 0x01d883060fddef01
Faulting application path: Anamnesis.exe
Faulting module path: C:\Program Files (x86)\RivaTuner Statistics Server\RTSSHooks64.dll
Report ID: 4f01a968-4498-452d-99e6-e27cb2fff406
Faulting package full name: 
Faulting package-relative application ID: `

`Application: Anamnesis.exe
CoreCLR Version: 6.0.622.26707
.NET Version: 6.0.6
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 000001BDEDB04FB0`

`Fault bucket 1966105318795382205, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Anamnesis.exe
P2: 1.0.1.0
P3: 62571213
P4: RTSSHooks64.dll
P5: 0.0.0.0
P6: 5aaa3d86
P7: c0000005
P8: 0000000000014fb0
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.79c11422-f695-4da3-ba33-067912db4dae.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.0002a328-c702-45a8-ae8b-5765705cfd29.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.3eacee16-76a1-4868-a53d-d217cce46fc2.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.42a99539-ad05-4029-bdb5-0c36317cb18d.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.86cde819-c3d2-4b11-96ed-14519403ed34.tmp.txt

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Anamnesis.exe_504d4ca7de899035da1e71ab7f8166335a9e9425_6aff99cd_4a09b005-4f14-4c07-b40f-7cfc421706d0

Analysis symbol: 
Rechecking for solution: 0
Report Id: 4f01a968-4498-452d-99e6-e27cb2fff406
Report Status: 268435456
Hashed bucket: f5837237c7b54928cb49025eddce0dbd
Cab Guid: 0`
